### PR TITLE
Add the tolerations for monitoring components section

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@ahardin-rh Hi, could you please review this?
(to be merged to enterprise-4.2)

File: modules/monitoring-assigning-tolerations-to-monitoring-components.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// * monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc
+
+[id="assigning-tolerations-to-monitoring-components_{context}"]
+= Assigning tolerations to monitoring components
+
+You can assign tolerations to any of the monitoring stack components to enable moving them to tainted nodes.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
```
Comment (ahardin-rh) [inline modules/monitoring-assigning-tolerations-to-monitoring-components.adoc]: s/Make sure/Ensure

**Comment by ahardin-rh** [comment]
@rh-max Just one minor comment from me. :+1: Once you submit a fix, please squash all commits into one and I will merge and cherry-pick. :rocket:

**Comment by rh-max** [comment]
@ahardin-rh Let me reject the suggestion, I use "Make sure" consistently throughout monitoring and changing it to "Ensure" everywhere wouldn't work from the grammar standpoint in some cases. And they mean the same thing, although "Ensure" is more formal.
Squashed. Thanks!

**Comment by ahardin-rh** [comment]
@rh-max Works for me. Thanks!

**Comment by rh-max** [comment]
@ahardin-rh You merged before I finish the comment :) Let me know if you insist on the ^ suggestion.

**Comment by rh-max** [comment]
@ahardin-rh Great!

File: modules/monitoring-assigning-tolerations-to-monitoring-components.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+data:
+  config.yaml: |
+    *_component_*:
+      tolerations:
+        *_tolerations specification_*
+----
++
+Substitute `<component>` and `<toleration specification>` accordingly.
++
+For example, a `kubectl taint nodes node1 key1=value1:NoSchedule` taint prevents the scheduler from placing pods in `foo: bar`. To make the `alertmanagerMain` component ignore that taint and to place `alertmanagerMain` in `foo: bar` normally, use this toleration:
```
Comment (juzhao) [inline modules/monitoring-assigning-tolerations-to-monitoring-components.adoc]: `kubectl taint nodes node1 key1=value1:NoSchedule`
=>
`oc adm taint nodes node1 key1=value1:NoSchedule`

prevents the scheduler from placing pods in `foo: bar`
=>
prevents the scheduler from placing pods in `foo: bar` node

Comment (rh-max) [reply modules/monitoring-assigning-tolerations-to-monitoring-components.adoc]: Implemented in #16978.

**Pull request merged**